go.mod

```diff
@@ -1,6 +1,6 @@
 module google.golang.org/genproto
 
-go 1.23.0
+go 1.24.0
```

Comment:
`go 1.24 `is better, it is more explicit that googleapis as a library doesn't care the go patch version. Adding the suffix ".0" force all applications which depend on the lib have to add the suffix ".0" as well.